.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "printWidth": 120,
+  "singleQuote": true
```

Comment:
What do you think about adding `trailingComma: all`. I prefer trailing commas because once a new line is added it's only one line in diff rather than two lines because of the extra comma.

Reply:
hmm, I actually tried this setting out before but be aware that this changes seemingly a lot more lines in this repo or in other words there are currently more cases without trailing commas vs cases with trailing commas in this repo.
Let me know if I should proceed still.

Reply:
just pushed a variant with trailing commas

Reply:
done

Reply:
IMO looks good. What do you think?

Reply:
ok looks good.